skbio/alignment/tests/test_tabular_msa.py

```diff
@@ -493,6 +493,43 @@ def test_reindex_non_hashable_keys(self):
 
         npt.assert_array_equal(msa.keys, keys)
 
+    def test_append(self):
+        msa = TabularMSA([DNA('ACGT'), DNA('TGCA')])
+        msa.append(DNA('AAAA'))
+        self.assertEqual(msa.shape, (3, 4))
+        # TODO: Hack to get last seq. once __getitem__ is implemented use
```

Comment:
You can use `self.assertEqual` to compare `msa` to an expected `TabularMSA` (and the shape assertion above can be removed too!).